### ISSUE TITLE
Improve Slack notifications readability

### ISSUE THREAD

**Issue by batianusrey** [airbytehq/airbyte#8239]
## Tell us about the problem you're trying to solve
We've configured our Airbyte deployment to send notifications to a Slack channel to get feedback from when the connections run. However, the message within the notification references the connectors name and version (example: BigQuery version 0.5.0) instead of the name of the source/destination which is the name one gives when configuring the source or destination.

This makes it practically impossible to know which connection succeeded or failed if you have different sources or destinations with the same connector.

![image](https://user-images.githubusercontent.com/2190947/143271893-4c6fa7d8-a8b3-404c-b81a-a634b6c8ef7b.png)

Also, it's not easy at a first glance to know if it succeeded or failed (one needs to read the message it self).

## Describe the solution you’d like
It would be good to reference the "human" name given to the source/destination instead of the type and version of the connector. So instead of `Your connection from Hubspot version 0.1.24 to BigQuery version 0.5.0 succeeded` it could be `Your connection from CRM (Hubspot version 0.1.24) to DWH CRM BigQuery (BigQuery version 0.5.0) succeeded`.

Overall, a small UI improvement to know if the connection failed or succeeded would be to enrich the Slack notification. Here is an example of a successful notification from our CI/CD pipeline as a reference:

![image](https://user-images.githubusercontent.com/2190947/143272860-08b77b75-6d0b-48f1-a79d-4e6653e0b143.png)

Red or green can be used to specify quickly if it succeeded or failed, and instead of `environment`, `commit` (in our case) could be `source name`, `source connector`, `destination name`, `destination connector` or something like that. Just throwing some ideas ;)

**Comment by GKTheOne** [comment]
Can we also allow changing the base url of links?
I've installed in Kubernetes and all the slack links are using the cluster internal service name/port instead of the ingress host/ssl-port.

**Comment by jasonmaddernstudylink** [comment]
This would be fantastic. I have essentially stopped looking at these because there's no context.

+1 on both the message improvement to human readable source/destination, and
+1 on being able to transpose the URL so it's not just host.docker.internal

**Comment by mjerzyk-surfer** [comment]
hey, is there anything happening on that one? I'd love to see some green/red colours when job succeed/fail and it would be definitely a great improvement for the readability

**Comment by nataliekwong** [comment]
Hey @mjerzyk-surfer! 

This is now updated as of [v0.50.48](https://github.com/airbytehq/airbyte/releases/tag/v0.50.48). We did not utilize different colors, but we've added more contextual information to the notification and rebuilt the formatting. 

![Screenshot 2024-02-23 at 3 24 20 PM](https://github.com/airbytehq/airbyte/assets/38087517/1b3c5fc5-7385-4b6a-85dd-ef6fba5d254d)

I'll close this issue for now, and we can create new issues to address additional improvements!
